ethcore/src/client/client.rs

```diff
@@ -1065,14 +1064,15 @@ impl Client {
 		};
 
 		self.block_header(id).and_then(|header| {
-			let db = self.state_db.read().boxed_clone();
-
+			let state_db = self.state_db.read();
```

Comment:
Previously it released the read lock instantly, right? Not that it matters much, but something to be aware of.

Reply:
right